_data/resources.yml

```diff
@@ -175,93 +198,163 @@
   why: "increases understanding of the historical and redemptive narrative of the entire Bible."
 
 
+
+###############################################################################
+# TOOLS                                                                       #
+###############################################################################
+
+
 #
 # TOOLS: Translation
 #
 
-# translationStudio (tS)
+# Autographa
+- name: Autographa
+  abbreviation:
+  category: tools
+  sub-category: translation
+  url: https://unfoldingword.org/autographa/
```

Comment:
@jag3773 am I correct that the links on the pages themselves should still be built like this: 

`href="{{ '/tq/' | prepend: site.baseurl }}"`